### DISCUSSION TITLE
Setup sentry with doc

### DISCUSSION THREAD

**Discussion by tgmarinho** [UnlyEd/next-right-now#381] [General]
Sentry provides this [guide](https://docs.sentry.io/platforms/javascript/guides/nextjs/)

And your set up using node and react packages of the sentry.

Have you tried to [folow this guide.](https://docs.sentry.io/platforms/javascript/guides/nextjs/)?

**Comment by tgmarinho** [comment]
Ops! I saw now https://github.com/UnlyEd/next-right-now/pull/370

**Reply by Vadorequest** [reply]
Yep, it's WIP, and so far it's not looking so great. Plenty of devs encounter issues with the @sentry/nextjs package, especially on the API side of things. In the current WIP branch, it's not working at all and crashes the whole server.
